src/sidebar.js

```diff
@@ -346,12 +346,17 @@ class Sidebar extends Component {
       }
     }
 
+    if (this.props.rootId) {
```

Comment:
Could you put the rootId in rootProps at line 247 instead so it mirrors rootClassName? I think that would be cleaner.